src/core/model_utils.py

```diff
@@ -592,10 +599,60 @@ def pre_save(self, model_instance, *args, **kwargs):
             return data
 
 
+class JanewayBleachFormField(BleachFormField):
+    """
+    An override of BleachFormField
+    to avoid the same unwanted effects that
+    JanewayBleachField avoids.
+    """
+
+    widget = TinyMCE
+
+    def to_python(self, value):
+        if value in self.empty_values:
+            return self.empty_value
+        return clean(value, **self.bleach_options)
+
+
+class MiniHTMLFormField(JanewayBleachFormField):
+    """
+    A form field to hold limited HTML phrasing content,
+    generally for use inline or on one line.
+    It uses a much smaller bleach allowlist
+    and loads by default with a minimal TinyMCE widget.
+    It is the default formfield used by JanewayBleachCharField.
+    """
+
+    def __init__(self, *args, **kwargs):
+        kwargs['allowed_tags'] = get_allowed_html_tags_minimal()
+        kwargs['allowed_attributes'] = get_allowed_attributes_minimal()
+        super().__init__(*args, **kwargs)
+        # Override the default TinyMCE widget with a minimal one
+        self.widget = TinyMCE(
+            mce_attrs={
+                'plugins': 'help code',
+                'menubar': '',
+                'forced_root_block': 'div',
+                'toolbar': 'help removeformat | undo redo | ' \
+                           'bold italic superscript subscript',
+                'height': '8rem',
+                'resize': True,
+                'elementpath': False,
+            }
```

Comment:
as above, we'd lose the benefits of composition here by ignoring the provided `widget` in `kwargs`

Reply:
This field is affected in the same way as the others, so I made it consistent as a direct override of `kwargs` so that at least it is consistent with the other two.